Update pet wyvern and DRG parameter increases

soupcans opened the pull request:
Got DRG update information from here: 
http://forum.square-enix.com/ffxi/threads/43912-dev1229-Job-Adjustments

Square Enix added DRG bonus effects of attack, defense, haste percentage increase based on wyvern leveling up 5 times. 4% increase per level up, maximum 20% attack/defense boost. 2% increase per level up for haste, maximum 10% job ability haste.

As for changing wyvern HP percentage increase to 6% per level instead of 5% and accuracy from 2 to 6 per level:
https://www.bg-wiki.com/bg/Wyvern_(Dragoon_Pet)

"For the Wyvern per parameter increase:
HP +6% (Cap: +30%)
Attack +4.6875% [12/256] (Cap: +23.4375% [60/256])
Accuracy +6 (Cap: +30)"

teschnei commented on scripts/globals/pets/wyvern.lua:
```diff
@@ -133,13 +133,17 @@ function onMobSpawn(mob)
+                master:addMod(dsp.mod.HASTE_ABILITY,20*diff)
             end
             pet:setLocalVar("wyvern_exp", prev_exp + exp)
+            mob:setLocalVar("level_Ups", diff)
```
this needs to be `mob:setLocalVar("level_Ups", mob:getLocalVar("level_Ups") + diff)`, or else it's going to keep overwriting with the wrong number

diff is just the number of levels that the wyvern just got, which will usually be 1 (unless you killed something that gave  more than 1 level of exp)

teschnei commented on scripts/globals/pets/wyvern.lua:
```diff
@@ -133,13 +133,17 @@ function onMobSpawn(mob)
+                master:addMod(dsp.mod.HASTE_ABILITY,20*diff)
             end
             pet:setLocalVar("wyvern_exp", prev_exp + exp)
+            mob:setLocalVar("level_Ups", mob:getLocalVar("level_Ups") + diff)
```
the entity you act on must be "pet" here for both setLocalVar and getLocalVar (while technically "mob" is captured from outside the scope, due to a bug in our lua engine, the entity will be deleted memory, and will probably crash)

teschnei replied:
sorry, didn't notice it last time :)

soupcans commented:
Ok, changed mob:setLocalVar to pet:setLocalVar

takhlaq commented on scripts/globals/pets/wyvern.lua:
```diff
@@ -148,8 +152,15 @@ end;
     local master = mob:getMaster();
+    local pet = player:getPet();
+    local numLvls = pet:getLocalVar("level_Ups");
+    if (numLvls ~= nil) then
```
`CLuBaseEntity:getLocalVar` returns 0 which i dont think is equivalent to `nil` in lua  (see: http://lua-users.org/wiki/ExpressionsTutorial)
```lua
if numLvls ~= 0 then
```
should do it

soupcans commented:
Updated with style corrections, deleted unnecessary semicolons and parentheses. Added spacing between operations and after comma. 

Changed numLvls ~= nil to numLvls ~=0

takhlaq commented:
thanks!
